django/thunderstore/repository/migrations/0031_add_namespaces_to_existing_packages.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.1.7 on 2021-11-04 20:32
+
+from django.db import migrations
+
+
+def forwards(apps, schema_editor):
+    Package = apps.get_model("repository", "Package")
+    packages = Package.objects.all()
```

Comment:
Always use `.iterator()` in migrations or you risk running out of memory during execution.

Reply:
That said, since you are using the `owner` relation, you should really do `.annotate(owner_name=F("owner__name"))` as well so you can use that field without an additional query. `select_related` doesn't work here as `.iterator()` disables queryset cache, but annotation will work.

Likewise you should then use `p.owner_name` instead of `p.owner.name` when referring to it to avoid the additional query.